src/main/java/taxi/controller/LogoutController.java

```diff
@@ -0,0 +1,17 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getSession().invalidate();
+        resp.sendRedirect(req.getContextPath() + "index");
+
```

Comment:
empty line + a suggestion 
```suggestion
        resp.sendRedirect(req.getContextPath() + "/login");
```

Reply:
amended